packages/cli/src/commands/list.ts

```diff
@@ -326,16 +322,23 @@ function getProjectName(d: Deployment) {
 
 // renders the state string
 export function stateString(s: string) {
-  switch (s) {
+  const CIRCLE = '● ';
+  // make `s` title case
+  s = `${s.substring(0, 1)}${s.toLowerCase().substring(1)}`;
```

Comment:
Because we need both values, I don't think we should change the value of `s`, just to temporarily revert it with `.toUpperCase`. When reading this logic, I feel like I'm missing some detail of why this is necessary.

How about leaving `s` as is and making a new variable to hold the titlecase version?

Reply:
TIL!